drivers/wifi/nrf700x/src/wifi_mgmt.c

```diff
@@ -158,6 +165,13 @@ int nrf_wifi_get_power_save_config(const struct device *dev,
 	}
 
 	rpu_ctx_zep = vif_ctx_zep->rpu_ctx_zep;
+
+	k_mutex_lock(&vif_ctx_zep->vif_lock, K_FOREVER);
+	if (!rpu_ctx_zep->rpu_ctx) {
+		LOG_DBG("%s: RPU context not initialized", __func__);
+		goto out;
+	}
+
```

Comment:
I think the above check should be moved below the check for rpu_ctx_zep. If rpu_ctx_zep is not initialized, this will cause a NULL dereference